contrib/fwupd.spec.in

```diff
@@ -234,8 +234,8 @@ fwupd wrapper for Qubes OS
 %if 0%{?enable_ci}
     --werror \
     -Db_sanitize=address,undefined \
-    -Dumockdev_tests=disabled \
 %endif
+    -Dumockdev_tests=disabled \
```

Comment:
can this be moved into `enable_tests` block?

Reply:
I tried this, and got:

    data/tests/meson.build:113:6: ERROR: Command `/usr/bin/python3 /builddir/build/BUILD/fwupd-2.0.0-build/fwupd-2.0.0/generate-build/unittest_inspector.py /builddir/build/BUILD/fwupd-2.0.0-build/fwupd-2.0.0/data/tests/fwupd_test.py` failed with status 1.

on every version and arch... :) I don't think LD_PRELOAD is going to work in an rpmbuild.